src/Components/Items/ItemPersistentData.tsx

```diff
@@ -105,35 +107,39 @@ export const ItemPersistentData = ({ item }: Props) => {
         rightLabel="Reason"
         rightText={item.fields["System.Reason"]}
       ></TwoColumn>
-      <Stack vertical>
-        <GreyTitle>Assignee</GreyTitle>
-        <Stack gap={5} align="center">
-          {avatar.data ? (
-            <img
-              src={`data:image/jpeg;base64,${avatar.data.value}`}
-              style={{ width: "20px", height: "20px", borderRadius: "100%" }}
-            />
-          ) : (
-            <Avatar />
-          )}
-          <P1>{item.fields["System.AssignedTo"].displayName}</P1>
+      {assignedTo && (
+        <Stack vertical>
+          <GreyTitle>Assignee</GreyTitle>
+          <Stack gap={5} align="center">
+            {avatar.data ? (
```

Comment:
Use `Avatar` component from `deskpro-ui` for this case. There is a case for missing data

Reply:
I did and it was broken, would refuse to load images like this.